sources/vault.move

```diff
@@ -0,0 +1,122 @@
+/// User can deposit/withdraw multiple fungible assets to/from different markets
+module playground::vault {
```

Comment:
Is this just an experiment? I'm not seeing how this fits into your Compound-like lending protocol